lib/catcher_game/main_scene.dart

```diff
@@ -112,33 +106,20 @@ class MainScene extends PositionComponent
 
   @override
   void update(double dt) {
-    if (game.status == CatcherGameStatus.playing) {
+    if (game.status == CatcherGameStatusType.playing) {
       super.update(dt);
-    } else if (game.status == CatcherGameStatus.pause) {
+    } else if (game.status == CatcherGameStatusType.pause) {
```

Comment:
I agree with your comment, but its sad to say but it would take some time for refactoring. I would keep this improvement in mind and add a todo for a later time

Reply:
[8cc0918](https://github.com/DenisovAV/recyclo/pull/19/commits/8cc0918b42a34090d214c917158e01ab2be1bf73)